src/AppInstallerCommonCore/AppInstallerStrings.cpp

```diff
@@ -105,6 +105,18 @@ namespace AppInstaller::Utility
         return ToLower(a) == ToLower(b);
     }
 
+    bool CaseInsensitiveContains(std::vector<std::string_view> a, std::string_view b)
+    {
+        for (std::string_view c : a)
+        {
+            if (CaseInsensitiveEquals(c, b))
+            {
+                return true;
+            }
+        }

```

Comment:
Not saying that it would be better, but FYI you can do this with `std::any_of`. Something like
```c++
return std::any_of(a.begin(), a.end(), [=](const std::string_view s) { return CaseInsensitiveEquals(s, b); });
```